README.md

```diff
@@ -1,70 +1,186 @@
+<!-- GETTING STARTED -->
 
-### Advanced Configuration
+## 💻 Getting Started <a name="getting-started"></a>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+-NodeJS installed.
+-Git installed.
+-A web browser of choice.
+
+### Setup
+
+Clone this repository to your desired folder:
+ 
+- Use the `git clone` command with this [link](https://github.com/Ellon-M/todo-peer-cr.git)
+- On the terminal, inside the desired folder's directory, run `npm install` to install all the necessary depandancies.
+- run `npm start` to execute the code and serve it on the browser.
+
+### Install
+
+- Install this project with:
+
+```
+$ npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```
+npm run start
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```
+npm run test
+```
 
 ### Deployment
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+You can deploy this project using:
+
+
+```
+npm run deploy
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Ellon M**
+
+- GitHub: [@githubhandle](https://github.com/Ellon-M)
+- Twitter: [@twitterhandle](https://twitter.com/ellonm4)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/ellon-m)
+
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+Coming soon
+
+ <p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+If you want to contribute to this project, create a fork and submit a pull request.
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please give it as star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for providing the necessary guidance and resources that helped in making this project successful.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+<!-- ## ❓ FAQ <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+LICENSE
+
+## 📝 License <a name="license"></a>
 
-### `npm run build` fails to minify
+This project is [MIT](./LICENSE) licensed.
```

Comment:
Kindly fix this issue of the link to the MIT which is not working. Watch the video at this link: 
https://www.loom.com/share/39c137bdbe5f40d68822086c8242d850

Please document your README to make the MIT license link work. Kindly use [this template](https://github.com/microverseinc/readme-template) and customize it to your project.